dropshot/src/router.rs

```diff
@@ -842,9 +842,11 @@ mod test {
             "project_id"
         ]);
         assert_eq!(result.variables.get("project_id").unwrap(), "p12345");
-        assert!(router
-            .lookup_route(&Method::GET, "/projects/p12345/child")
-            .is_err());
+        assert!(
+            router
+                .lookup_route(&Method::GET, "/projects/p12345/child")
+                .is_err()
+        );
```

Comment:
I'm guessing this is the change youre not a fan of? because the other one actually looks good to me!

Reply:
Meh. I'm neutral on this one.

Reply:
ah, i mis-read `handler.rs` as `router.rs`, oops 😓 

(I am also neutral tbh)